quest_test/test_persistence.py

```diff
@@ -1,13 +1,33 @@
 import asyncio
 from pathlib import Path
+from typing import Callable
+from dotenv import load_dotenv # TODO: Remove, how should we load env in the test?
 
 import pytest
 
-from src.quest import step
-from src.quest.historian import Historian
-from src.quest.persistence import PersistentHistory, LocalFileSystemBlobStorage
+from quest import step
+from quest.historian import Historian
+from quest.persistence import PersistentHistory, LocalFileSystemBlobStorage, SQLDatabase, DynamoDB, SqlBlobStorage, DynamoDBBlobStorage
 from utils import timeout
 
+def create_filesystem_storage(path: Path):
+    return LocalFileSystemBlobStorage(path)
+
+def create_sql_storage(path: Path):
+    database = SQLDatabase('sqlite:///:memory:')
+    return SqlBlobStorage(path.name, database.get_session())
+
+def create_dynamodb_storage(path: Path):
+    env_path = Path('../.env')
+    load_dotenv(dotenv_path=env_path)
```

Comment:
I am using dotenv here just for testing purposes. Do we want to mock this functionality somehow? Any suggestions?